### PR TITLE
chore: LenderMarket & BorrowerMarket table populated using data from created proposal

### PR DESCRIPTION
**Pull request by DanielEmmanuel1**
Implemented functionality to populate the LenderMarket and BorrowerMarket tables dynamically with data from created proposals. This enhancement ensures that user-generated proposals are seamlessly integrated into the platform's marketplace, allowing lenders and borrowers to view and interact with active proposals directly through the market interface. This feature enhances user engagement by providing real-time visibility into available lending and borrowing opportunities, thereby facilitating efficient peer-to-peer transactions within the Peer Protocol ecosystem

## Summary by CodeRabbit

- **New Features**
	- Updated text in the Hero Section for improved clarity.
	- Enhanced Borrowers and Lenders Market components to allow dynamic proposal creation and management.
	- Improved user interface with new input fields and better display for proposals.

- **Bug Fixes**
	- Updated styling in the Footer for a semi-transparent effect.

- **Chores**
	- Added commented-out links in the Sidebar for future use.

### CODE REVIEW

**Review by Peerprotocol — CHANGES_REQUESTED** [review]
please can we work on these suggested changes by codeRabbit AI

File: src/app/app/BorrowMarket/page.tsx [hunk 1]
```diff
@@ -189,7 +222,10 @@ const BorrowersMarket = () => {
                 <label className="text-sm text-gray-500 pl-2">Quantity</label>
                 <div className="p-3 border rounded-xl border-gray-600">
                   <input
-                    type="text"
+                    type="number"
+                    name="quantity"
+                    value={newProposal.quantity}
+                    onChange={handleInputChange}
```
[the remark this answers is not in the document]
Comment (DanielEmmanuel1) [reply src/app/app/BorrowMarket/page.tsx:228]: The netvalue will be automatically calculated, its not supposed to be an inputed value, its supposed to be the quantity of the token multiplied by the price of 1 unit of the token netValue = tokenQuantity * pricePerUnitToken

**Comment by DanielEmmanuel1** [comment]
All done @Peerprotocol

**Comment by Dprof-in-tech** [comment]
Alright
I’ll look through in a bit

**Review by Peerprotocol — APPROVED** [review]

**Pull request merged**